### DISCUSSION TITLE
Varying J2000.0 AR/Dec values over time

### DISCUSSION THREAD

**Discussion by diegorodriguezperegrin** [Stellarium/stellarium#2456] [Q&A]
I undestand that in Stellarium J2000.0 AR/Dec do not change over time, except some cyclic variation due to leap years and proper motions. I also asume that nutation and precession do only come into effect when calculating coordinates on date. Is this so? Many thanks for clarifying!

**Comment by gzotti** [comment]
https://en.wikipedia.org/wiki/Epoch_(astronomy)#J2000

**Comment by diegorodriguezperegrin** [comment]
Thank you. I have read carefully this entry, but I'd like to know what does Stellarium to bring J2000.0 AR/Dec into Coordinates on a particular date. I can imagine that Stellarium adds nutation, precession, parallax and proper motion. Nevertheless, even within J2000.0, I see J2000.0 coordinates for a particular remote object changing over long periods of time. So why do they?

**Reply by gzotti** [reply]
Most likely proper motion.
